docs/resources/rms_originational_assignment_package.md

```diff
@@ -0,0 +1,103 @@
+  name         = "assignment_package_name"
+  template_key = "organizational_assignment_package_test.tf.json"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `organization_id` - (Required, String, ForceNew) Specifies the ID of the organization.
+
+  Changing this parameter will create a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the assignment package name. It contains 1 to 64 characters.
+
+  Changing this parameter will create a new resource.
+
+* `excluded_accounts` - (Optional, List, ForceNew) Specifies the excluded accounts for conformance package deployment.
+
+  Changing this parameter will create a new resource.
+
+* `template_key` - (Optional, String, ForceNew) Specifies the name of a predefined conformance package. It contains 1 to
+  128 characters.
+
+  Changing this parameter will create a new resource.
+
+* `template_body` - (Optional, String, ForceNew) Specifies the content of a custom assignment package. It contains 1 to
+  51200 characters.
+
+  Changing this parameter will create a new resource.
+
+* `template_uri` - (Optional, String, ForceNew) Specifies the OBS address of a conformance package. It contains 1 to
+  1024 characters.
+
+  Changing this parameter will create a new resource.
+
+* `vars_structure` - (Optional, List, ForceNew) Specifies the parameters of a conformance package.
+
+  Changing this parameter will create a new resource.
+The [vars_structure](#OrgAssignmentPackage_VarStructure) structure is documented below.
+
+<a name="OrgAssignmentPackage_VarStructure"></a>
+The `vars_structure` block supports:
+
+* `var_key` - (Optional, String) Specifies the name of a parameter. It contains 1 to 64 characters.
+
+* `var_value` - (Optional, String) Specifies the value of a parameter.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID.
+
+* `owner_id` - Indicates the creator of an organization conformance package.
+
+* `org_conformance_pack_urn` - Indicates the unique identifier of resources in an organization conformance package.
+
+* `created_at` - Indicates the creation time of an organization conformance package.
+
+* `updated_at` - Indicates the latest update time of an organization conformance package.
+
+## Import
+
+The RMS organizational assignment package can be imported using the `organization_id` and `id` separated by a slash, e.g.
+
+```bash
+$ terraform import huaweicloud_rms_originational_assignment_package.test <organization_id>/<id>
+```
+
+Note that the imported state may not be identical to your resource definition, due to some attributes missing from the
+API response, security or some other reason. The missing attributes include: `agency_name`, `template_key`,
```

Comment:
there is none parameters named `agency_name`

Reply:
done